cmd/hope/vm/create.go

```diff
@@ -0,0 +1,120 @@
+package vm
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+)
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+import (
+	"github.com/Eagerod/hope/cmd/hope/utils"
+	"github.com/Eagerod/hope/pkg/packer"
+	"github.com/Eagerod/hope/pkg/ssh"
+)
+
+var createCmdVmName string
+var createCmdMemory string
+var createCmdCpu string
+
+func initCreateCmdFlags() {
+	createCmd.Flags().StringVarP(&createCmdVmName, "name", "n", "", "what to name the virtual machine. If left blank, defaults to the hope file name.")
+	createCmd.Flags().StringVarP(&createCmdMemory, "memory", "m", "", "how much memory to given the created VM.")
+	createCmd.Flags().StringVarP(&createCmdCpu, "cpu", "c", "", "how many vCPUs to given the created VM.")
+}
+
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Creates a VM on the specified host.",
+	Args:  cobra.ExactArgs(2),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		hypervisorName := args[0]
+		vmName := args[1]
+
+		if createCmdVmName == "" {
+			return errors.New("Must provide a VM name")
+		}
+
+		hypervisor, err := utils.GetNode(hypervisorName)
```

Comment:
Feels like there's a lot of repeated validation/parameter management around fetching nodes/vm specs etc. Should figure out a way to keep that all consolidated at some point.